download.sh

```diff
@@ -64,7 +67,7 @@ function install {
 	#choose from asset list
 	URL=""
 	FTYPE=""
-	DEFAULT_VERSION="v1.4.0"
+	DEFAULT_VERSION=$(curl -L https://api.github.com/repos/blessnetwork/bls-javy/releases/latest|jq -r '.tag_name')
```

Comment:
nit: spaces between pipe operator
```suggestion
	DEFAULT_VERSION=$(curl -L https://api.github.com/repos/blessnetwork/bls-javy/releases/latest | jq -r '.tag_name')
```